lib/curator.rb

```diff
@@ -0,0 +1,56 @@
+class Curator
+  attr_reader :photographs, :artists
+
+  def initialize
+    @photographs = []
+    @artists = []
+  end
+
+
+  def add_photograph(photo)
+    @photographs << photo
+  end
+
+  def add_artist(artist)
+    @artists << artist
+  end
+
+  def find_artist_by_id(id)
+    found = []
+    @artists.each do |artist|
+      found << artist if artist.id == id
+      end
+    @found
+  end
+
+  def find_photograph_by_id(id)
+    found = []
+    @photographs.each do |photo|
+      found << photo if photo.id == id
+    end
+    found
+  end
+
+  def find_photographs_by_artist(artist)
+    return_photos = []
+    @photographs.each do |photo|
+      return_photos << photo if photo.artist_id  == artist.id
+    end
+    return_photos
```

Comment:
as a tip, any time you're doing a .each with an if statement within it, you can probably consider find or find_all to do what you're looking for